argilla/docs/tutorials/image_classification.ipynb

```diff
@@ -0,0 +1,804 @@
+{
```

Comment:
<p>Duplicated: "As we can see"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>